### PR TITLE
worked on student fees module class

### PR DESCRIPTION
**Pull request by Richmond-Andoh**

### CODE REVIEW

File: CorsaSchoolManagementSystem/Views/studentDetails.php [hunk 1]
```diff
@@ -334,95 +334,52 @@
...
-                  <label class="fw-bold py-2">Date of Payment</label>
-                  <input type="text" name="dateOfPayment" class="form-control">
+                  <label class="fw-bold py-2">Academic Year</label>
+                  <input type="text" name="academicYear" class="form-control">
                 </div>
                 <div class="container my-3">
-                  <label class="fw-bold py-2">Status</label>
-                  <input type="text" name="status" class="form-control">
+                  <label class="fw-bold py-2">Date of Payment</label>
+                  <input type="text" name="dateOfPayment" class="form-control">
```
Comment (OliverMensahDev) [inline CorsaSchoolManagementSystem/Views/studentDetails.php]:
```suggestion
                  <input type="date" name="dateOfPayment" class="form-control">
```

Comment (Richmond-Andoh) [reply CorsaSchoolManagementSystem/Views/studentDetails.php]: Alright

File: CorsaSchoolManagementSystem/Models/StudentFees [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?php
+ require_once("Core/init.php");
```
Comment (OliverMensahDev) [inline CorsaSchoolManagementSystem/Models/StudentFees]:
```suggestion

```

**Review by OliverMensahDev — COMMENTED** [review]
Remove that line of code

**Comment by Richmond-Andoh** [comment]
sure

**Review by OliverMensahDev — APPROVED** [review]

**Pull request merged**